plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseCaseInsensitiveMappingTest.java

```diff
@@ -90,11 +88,9 @@ public void testNonLowerCaseTableName()
             assertQuery(
                     "SELECT column_name FROM information_schema.columns WHERE table_name = 'nonlowercasetable'",
                     "VALUES 'lower_case_name', 'mixed_case_name', 'upper_case_name'");
-            assertEquals(
-                    computeActual("SHOW COLUMNS FROM someschema.nonlowercasetable").getMaterializedRows().stream()
+            assertThat(computeActual("SHOW COLUMNS FROM someschema.nonlowercasetable").getMaterializedRows().stream()
                             .map(row -> row.getField(0))
-                            .collect(toImmutableSet()),
-                    ImmutableSet.of("lower_case_name", "mixed_case_name", "upper_case_name"));
+                            .collect(toImmutableSet())).containsOnly("lower_case_name", "mixed_case_name", "upper_case_name");
```

Comment:
```suggestion
                            .collect(toImmutableSet()))
                            .containsOnly("lower_case_name", "mixed_case_name", "upper_case_name");
```

reindent in IDE, I was lazy

Reply:
How about using `extracting` instead?
```java
            assertThat(computeActual("SHOW COLUMNS FROM someschema.nonlowercasetable"))
                    .extracting(row -> row.getField(0))
                    .containsOnly("lower_case_name", "mixed_case_name", "upper_case_name");
```